node/impl/full/gas.go

```diff
@@ -248,22 +248,19 @@ func (m *GasModule) GasEstimateGasLimit(ctx context.Context, msgIn *types.Messag
 	}
 	return gasEstimateGasLimit(ctx, m.Chain, m.Stmgr, m.Mpool, msgIn, ts)
 }
-func gasEstimateGasLimit(
+
+func gasEstimateCallWithGas(
```

Comment:
Can we add a comment describing the return here (specifically the `priorMsgs` bit).